package.json

```diff
@@ -13,6 +13,7 @@
     "web-vitals": "^2.1.3"
   },
   "scripts": {
+    "format": "npx prettier --write .",
```

Comment:
You might not need to use `npx prettier --write .` here since you are installing it as a `devDependency` of your project! See if you can make it run without it, just `prettier --write .`

Reply:
fixed